test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -488,3 +505,13 @@ def test_mom_restart(self):
                            interval=2)
         self.server.log_match(jid + ";Exit_status=0", interval=4,
                               max_attempts=10)
+
+    def tearDown(self):
+        for mom_val in self.moms.values():
+            if mom_val.is_cpuset_mom():
+                mom_val.restart()
```

Comment:
instead of restart, can we try deleting the directories. would it be an issue?
@bayucan - could you please guide.

Reply:
@prakashcv13 my thinking is that let PBS handle itself to delete those directories by restarting mom.

Reply:
@prakashcv13 : we can also have ptl delete the directories, but it's okay too to just let PBS handle that via mom restart.

Reply:
thank you @bayucan and @dinesh1306